en/debug/probe_mculink.md

```diff
@@ -0,0 +1,83 @@
+# MCU-Link Debug Probe
+
+The [MCU-Link Debug Probe](https://www.nxp.com/design/design-center/software/development-software/mcuxpresso-software-and-tools-/mcu-link-debug-probe:MCU-LINK) is a cheap, fast and highly capable debug probe that can serve as a stand-alone debug and console communicator whn working with Pixhawk boards.
+
+Key features:
+
+- Just one single USB-C connection for Reset, SWD, SWO, and serial in a very small package!
+- Up to 9.6MBit/s SWO connection.
+  Up to 5 MBaud serial. 1.2V to 5V target voltage.
+  USB2 high-speed 480 Mbps connection.
+- Driven by NXP LinkServer or pyOCD software with wide device support.
+- Much cheaper (<15€) than a Pixhawk Debug Adapter (~20€) with a JLink EDU mini (~55€) or JLink BASE (~400€) while having better hardware specs.
+
+The [Pixhawk Debug Adapter](https://holybro.com/products/pixhawk-debug-adapter) provides an easy way to connect a Pixhawk to an MCU-Link (the probe does not come with an adapter for working with Pixhawk flight controllers).
+
+:::note
+These instructions have been tested on: FMUv6X-RT, FMUv6X, FMUv6c, FMUv5X.
```

Comment:
I presume it works on these boards based on your comment (you didn't mention 6x, but that is mentioned lower down)